packages/api/utils/businessRulesValidation.test.ts

```diff
@@ -0,0 +1,9 @@
+import { uniqueNameValidator } from "./businessRulesValidation";
+
+describe("uniqueNameValidator", function () {
+  it("should throw an error if name is not unique", async () => {
+    const name = "unique name";
+    const names = ["not unique", "unique name"];
```

Comment:
The test is effective, but I think the specific values are confusing.
```suggestion
    const name = "orange";
    const names = ["apple", "orange"];
```

Reply:
Yeah, good point.